modules/hcp-deploy-openstack-parameters.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/hcp-deploy/hcp-deploy-openstack.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="hcp-deploy-openstack-parameters_{context}"]
+= Options for creating a Hosted Control Planes cluster on OpenStack
+
+You can supply several options to the `hcp` CLI while deploying a Hosted Control Planes Cluster on {rh-openstack-first}.
+
+|===
+|Option|Description|Required
+
+|`--openstack-ca-cert-file`
+|Path to the OpenStack CA certificate file.
```

Comment:
If this is not provided, it will be automatically pulled from the `clouds.yaml` (since https://github.com/openshift/hypershift/pull/4848). We will likely deprecate this option in the future.

```suggestion
|Path to the OpenStack CA certificate file. If not provided, this will be automatically extracted from the cloud entry in `clouds.yaml`.
```